notification/src/models/notificationSummary.ts

```diff
@@ -0,0 +1,16 @@
+import { WebHookConfig, SlackConfig, FirebaseConfig } from './notificationConfig';
+
+export class NotificationSummary{
```

Comment:
Do we need a class, or is an interface enough? (you can use interfaces and instate them with `= {attr1: xy, attrb: yz, ...}` instead of using a constructor).
Interfaces are typically used for pure data models without additional functionality, only for holding data.

Reply:
Interface for this entity should be enough. I  will change this today